aten/src/ATen/core/ivalue.cpp

```diff
@@ -80,4 +80,8 @@ std::ostream& operator<<(std::ostream & out, const IValue & v) {
 
 #undef TORCH_FORALL_TAGS
 
+void IValue::dump() const {
```

Comment:
We should standardize dump as an api for printing out objects... Sometimes it's toStr() etc

Reply:
I am for `dump()`, since that's what nodes and graphs use already.